src/adapters/github.js

```diff
@@ -146,10 +146,13 @@ class GitHub extends PjaxAdapter {
       return cb();
     }
 
+    // Trivial check whether typeId is a commit id in case URL '.../tree/{commitId}'
```

Comment:
in case URL '.../commit/{commitId}'

Reply:
I mean this case https://github.com/ovity/octotree/tree/b06e421795b403d46939055c30a5c1e1b111c706